### DISCUSSION TITLE
Feature: Weekly Agenda

### DISCUSSION THREAD

**Discussion by ghost** [Etar-Group/Etar-Calendar#1167] [Ideas]
Hi there!
Is there a chance for a different approach to a weekly calandar view?
I suggest an overview showing all 7 days of the week and appointments per day as a list with times, similar to the way acalendar (screenshot appended) shows the week, 
Let me sketch it: It's similar to a two column table with 4 rows (makes 8 fields),

Mo |  Fr
Tu  |  Sa
We |  Su
Th  |  (full month overview or so)

each field showing appointments in a simple list like this:
11:00 second breakfast

11:30 have a nap
12:00 lunnch

Unfortunately, I am totaly uncapable of doing it  myself...

![unnamed](https://user-images.githubusercontent.com/57146522/170472450-6ecf461f-1247-48d5-9e17-44b4bec9bdc5.png)

**Comment by marbx** [comment]
Hi @xebeque ,
This is a nice mockup and I would appreciate it too, if the days of the week were arranged as squares next and below to each other, like on on chessboard.

You will find that my proposal https://github.com/Etar-Group/Etar-Calendar/discussions/1005 is arranging the squares in a different way that you propose. Essentially, I split the square for Saturday and Sunday. This is how the Palm Pilot presented the week.

The fundamental problem is to arrange days as squares. I am unable to locate the piece of code that arranges days, but I intent to continue looking.
